spec/bundler/definition_spec.rb

```diff
@@ -132,6 +132,32 @@
            #{Bundler::VERSION}
       G
     end
+
+    context "when replacing a host with the same host with credentials" do
```

Comment:
Would it be better on `commands/install_spec.rb` on a `deployment` context?
It seems its separated by commands and `deployment` seems like a modifier to `install` not a command on its own